terraform/aws-multi-cluster/crawler-cfn.yml

```diff
@@ -50,7 +50,7 @@ Resources:
                 Action:
                   - 's3:GetObject'
                   - 's3:PutObject'
-                Resource: !Sub 'arn:$${AWS::Partition}:s3:::${CUR_BUCKET}/${CUR_PREFIX}/${ACCOUNT_NUM}*'
+                Resource: !Sub 'arn:$${AWS::Partition}:s3:::${CUR_BUCKET}/${CUR_PREFIX}/*'
```

Comment:
@falgofrancis Have you confirmed this works in your environment?

Reply:
yes, we have CUR data from single folder.